File: medplum-java/medplum-model/src/main/java/com/fhir/Patient_Link.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutablePatient_Link.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface Patient_Link {
  Optional<String> id();

  Optional<List<Extension>> extension();

  Reference other();

  Optional<Patient_linkType> type();

  Optional<List<Extension>> modifierExtension();

  static ImmutablePatient_Link.OtherBuildStage builder() {
    return ImmutablePatient_Link.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/Patient_Link.ts
import { Extension } from "./Extension";
import { Patient_linkType } from "./Patient_linkType";
import { Reference } from "./Reference";

export interface Patient_Link {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    other: Reference;
    type: Patient_linkType | null | undefined;
}
